featherbed-core/src/main/tut/02-basic-usage.md

```diff
@@ -30,7 +30,8 @@ Create a `Client`, passing the base URL to the REST endpoints:
 
 ```tut:book
 import java.net.URL
-val client = new featherbed.Client(new URL("http://localhost:8765/api/"))
+import featherbed.Client
```

Comment:
Not sure I agree with this. There are dozens of things called `Client` in finagle, so to me it's clearer to explicitly identify it.